docs/src/main/sphinx/release/release-454.md

```diff
@@ -0,0 +1,83 @@
+# Release 454 (7 Aug 2024)
+
+## General
+
+* Add support for retrieving lightweight query information from the `/v1/query`
+  endpoint with the new `pruned` property. ({issue}`20843`)
+* Improve performance of queries with multiple aggregations involving
+  `DISTINCT`. ({issue}`22355`)
+* Improve server stability by discarding query completed events when event
```

Comment:
"The maximum number pending events before new events are discarded can be configured with the ..."